dev/src/types.ts

```diff
@@ -199,6 +199,14 @@ export interface Settings {
    */
   maxIdleChannels?: number;
 
+  /**
+   * Whether to use `BigInt` for integer types when deserializing Firestore
+   * Documents. Regardless of magnitude, all integer values will be returned as
+   * `BigInt` to match the precision of the Firestore backend. Floating point
+   * numbers continue to use JavaScript's `number` type.
```

Comment:
maybe s/continue to/will for consistent future tense?

Reply:
Opted to change this to present tense for both.